### PR TITLE
20220329 config json

### PR DESCRIPTION
**Pull request by kenskyfish**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -293,7 +297,7 @@
         },
         {
           "name": "network",
-          "viewName": "VPC Network Name *",
```
Comment (graddaniel) [inline config.json:296]: Shouldn't this one receive `required` property?

File: config.json [hunk 2]
```diff
@@ -772,13 +784,13 @@
...
           "autocompleteType": "function",
           "functionName": "listProjectsAuto",
-          "description": "GCP Project defines how your app interacts with services and resources",
+          "description": "Google Cloud project forms the basis for creating, enabling, and using all Google Cloud services including managing APIs and managing permissions for Google Cloud resources",
           "placeholder": "select using autocomplete",
           "learnUrl": "https://cloud.google.com/resource-manager/docs/creating-managing-projects"
         },
         {
           "name": "network",
-          "viewName": "VPC Network Name *",
```
Comment (graddaniel) [inline config.json:781]: and this one?

File: config.json (hunk 1 above)
Comment (kenskyfish) [reply config.json:296]: No. It is useful for narrowing the autocomplete search for subnets, but left blank the method still works.

File: config.json (hunk 2 above)
Comment (kenskyfish) [reply config.json:781]: No. It is useful for narrowing the autocomplete search for subnets, but left blank the method still works.

**Review by graddaniel — DISMISSED** [review]

**Review by graddaniel — DISMISSED** [review]

**Comment by kenskyfish** [comment]
Too old, abandoning these cosmetic changes.

**Pull request closed without merging**